src/main.cpp

```diff
@@ -148,6 +151,9 @@ int main(int argc, char *argv[])
   {
     switch (c)
     {
+      case 2000:
+        std::cerr << BPFfeature().report();
+        return 1;
```

Comment:
Why return 1?

Reply:
I copied the wrong one, I'll fixi t

Reply:
Cause I copied wrong, I'll update it.

Reply:
this misses the root check  too,  neeeds to  be moved down